recipe/meta.yaml

```diff
@@ -76,6 +78,8 @@ requirements:
     - curl
     - zlib
     - hdf4
+    # without this repeat reference, conda-smithy doesn't pin correctly
+    - hdf5
```

Comment:
Looking at the docs, it looks like the redundancy is only needed in `host`, not `run`.  At the risk of restarting CI, I'm going to fix that.

Reply:
sure whatever you think is best!

Reply:
make sure to rerender

Reply:
Yep, done!